src/formatDate.js

```diff
@@ -50,7 +50,35 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat[fromFormat.length - 1];
+  let newDate = date.split(fromSeparator);
+  const toSeparator = toFormat[toFormat.length - 1];
+  const arr = [];
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    for (let j = 0; j < fromFormat.length - 1; j++) {
+      if (toFormat[i].includes('Y') && fromFormat[i].includes('Y')) {
```

Comment:
You have a typo in your condition, it should be `fromFormat[j]` instead of `fromFormat[i]`.